easybuild/easyconfigs/g/GenotypeHarmonizer/GenotypeHarmonizer-1.4.14-Java-1.7.0_80.eb

```diff
@@ -14,6 +14,8 @@ javaver = '1.7.0_80'
 versionsuffix = '-%s-%s' % (java, javaver)
 dependencies = [(java, javaver)]
 
+toolchain = {'name': 'dummy', 'version': 'dummy'}
```

Comment:
this should not be here, `toolchain` is already defined higher up in this easyconfig